src/index.js

```diff
@@ -418,25 +423,31 @@ ahoy.trackView = function (additionalProperties) {
 };
 
 ahoy.trackClicks = function () {
+  // <a id="link" href="/foo"><p id="text">Foo</p></a>
+  // e.target on Chrome (and likely every other browser) is not a#link tag but p#text 
+  // onEvent will callback with this = a#text and e.target = p#text
   onEvent("click", "a, button, input[type=submit]", function (e) {
-    let target = e.target;
-    let properties = eventProperties(e);
+    let target = this;
+    let properties = elementProperties(this); 
     properties.text = properties.tag == "input" ? target.value : (target.textContent || target.innerText || target.innerHTML).replace(/[\s\r\n]+/g, " ").trim();
     properties.href = target.href;
+    if(e.target != this){
+      properties.clickedElement = elementProperties(e.target);
```

Comment:
Let's avoid extra properties (users will be able to add them with #55 if needed).